gemfile.d/lock.rb

```diff
@@ -0,0 +1,2 @@
+gem "foreman_remote_execution", "< 13"
+gem "foreman_ansible", "< 14"
```

Comment:
For CI you can also provide an environment variable like https://github.com/dm-drogeriemarkt/foreman_wreckingball/blob/861f54cab044dafddefffe9e0402b9aeb330ba80/.github/workflows/ci.yml#L34-L35

Then you can use `ENV['FOREMAN_VERSION']` in this file. That can be useful to be more future compatible: when you start to support newer versions then you don't want to forget about this.

I'm using https://github.com/dm-drogeriemarkt/foreman_wreckingball/pull/116 to verify this theory. If so, I'll properly document it.